coderadar-test/src/test/java/io/reflectoring/coderadar/rest/query/GetMetricValuesOfTwoCommitsControllerTest.java

```diff
@@ -0,0 +1,180 @@
+package io.reflectoring.coderadar.rest.query;
```

Comment:
This file was the same as GetDeltaTreeForTwoCommitsControllerTest so I've deleted it when merging with master.